index.js

```diff
@@ -46,13 +46,13 @@ function runSequence(gulp) {
 		callBack = typeof taskSets[taskSets.length-1] === 'function' ? taskSets.pop() : false,
 		currentTaskSet,
 
-		finish = function(err) {
+		finish = function(e) {
 			gulp.removeListener('task_stop', onTaskEnd);
 			gulp.removeListener('task_err', onError);
 			if(callBack) {
-				callBack(err);
-			} else if(err) {
-				console.log(colors.red('Error running task sequence:'), err);
+				callBack(e.err);
+			} else if(e.err) {
```

Comment:
In both cases, this should be a little more robust, please change `e.err` to `e && e.err`.